apps/login/src/lib/server/loginname.ts

```diff
@@ -160,10 +162,52 @@ export async function sendLoginname(command: SendLoginnameCommand) {
     );
 
     if (!methods.authMethodTypes || !methods.authMethodTypes.length) {
-      return {
-        error:
-          "User has no available authentication methods. Contact your administrator to setup authentication for the requested user.",
-      };
+      if (
+        users.result[0].type.case === "human" &&
+        users.result[0].type.value.email &&
+        !users.result[0].type.value.email.isVerified
+      ) {
+        const paramsVerify = new URLSearchParams({
+          loginName: session.factors?.user?.loginName,
+          userId: session.factors?.user?.id, // verify needs user id
+        });
+
+        if (command.organization || session.factors?.user?.organizationId) {
+          paramsVerify.append(
+            "organization",
+            command.organization ?? session.factors?.user?.organizationId,
+          );
+        }
+
+        if (command.authRequestId) {
+          paramsVerify.append("authRequestId", command.authRequestId);
+        }
+
+        redirect("/verify?" + paramsVerify);
+      }
+      // what to do with users with valid email but no auth methods? redirect to /authenticator/set?
+      // return {
+      //   error:
+      //     "User has no available authentication methods. Contact your administrator to setup authentication for the requested user.",
+      // };
```

Comment:
What to do?

Reply:
removed the comment as the behavior is defined below